README.md

```diff
@@ -0,0 +1,239 @@
+
+After this, XBlock-poll may be installed using its setup.py, or if you prefer to use pip, running:
+
+    pip install /path/to/xblock_poll/repo/clone
+
+You may specify the `-e` flag if you intend to develop on the repo.
+
+### Setting up a course to use Polls and Surveys
+
+To set up a course to use the Poll and Survey XBlocks, first go to your course's outline page in the studio and look
+for the Advanced settings link in the top menus.
+
+![Advanced Settings](doc_img/advanced_settings.png)
+
+Once there, look for the *Advanced Modules List* and add `"poll"` and `"survey"` to it.
+
+![Advanced modules configuration](doc_img/advanced_modules_list.png)
+
+Save your changes, and you may now add a poll by clicking on the **Advanced Modules** button on the bottom of a
+unit editing page and selecting either `Poll` or `Survey`.
+
+## Poll Examples
+
+Polls contain a single question, letting users select from an array of answers for a choice that suits them best.
+
+Below is an example of a poll:
+
+![Poll example](doc_img/poll.png)
+
+When a user makes a selection and hits `submit`, their choice is sent to the server, and a tally of the results so far
+is presented to them.
+
+![Poll example results](doc_img/poll_result.png)
+
+The top choice's percentage is shown in *orange* while the user's selection is marked by a selected (but disabled)
+radio button on the side.
+
+### Poll variations
+
+Polls may have several customizations. Feedback may be added to a poll for display after the user has submitted their
+answer. Answers may have images associated with them. The question, feedback, and answers are all permitted to contain
+[Markdown](http://daringfireball.net/projects/markdown/syntax) (as well as arbitrary HTML) and so may be customized in the studio (as discussed in the next chapter).
+
+Here is an example of a poll that uses markdown in the question and the answers, and which uses images for each
+of the answers:
+
+![Image-only poll](doc_img/img_poll.png)
+
+This poll also contains a feedback section, which is enhanced with Markdown:
+
+![Image-only poll results](doc_img/img_poll_result.png)
+
+Polls may also have images and with text.
+
+![Image and Label label poll](doc_img/img_and_label_poll.png)
+
+Or they may have a mix of both.
+
+![Mixed label poll](doc_img/poll_mixed.png)
+
+Polls that have a combination of both images and text will autoformat their results so that the bars line up.
+
+![Mixed label poll result](doc_img/poll_mixed_result.png)
+
+## Survey Examples
+
+A survey has multiple answers and multiple questions. The same answers are presented for each question.
+
+Below is an example of a survey:
+
+![Survey example](doc_img/survey.png)
+
+When the user hits `submit`, their answer is sent to the server for tally. The results are then returned to them.
+
+![Survey results](doc_img/survey_result.png)
+
+The top choice's percentage is shown in *orange* while the user's selection is marked with a shaded background.
```

Comment:
I'm concerned about the color choices here.

Also the table is really squashed. Can we add more padding to the cells and especially add height padding?